pyqt-apps/siriushla/as_rf_control/util.py

```diff
@@ -0,0 +1,272 @@
+
+SEC_2_CHANNELS = {
+    'BO': {
+        'Emergency': 'BO-05D:RF-Intlk:EStop-Mon',
+        'Sirius Intlk': 'BO-05D:RF-Intlk:BO-Mon',
+        'LLRF Intlk': 'BO-05D:RF-LLRF:Status-Mon',
+        'Intlk Details': {
+            'Input': {
+                'PV': 'BR-RF-DLLRF-01:Intlk-Mon',
```

Comment:
I think all PV database information should eventually be retrieved from _siriuspy.csdevice_.

Reply:
I think we can move all PV database information to _siriuspy.csdevice_ when these RF PVs are adapted to our naming system...

Reply:
ok